jumpcloud-ADMU/Powershell/Start-Migration.ps1

```diff
@@ -2917,19 +3011,20 @@ Function Start-Migration {
         }
         if ([System.String]::IsNullOrEmpty($($admuTracker.Keys | Where-Object { $admuTracker[$_].fail -eq $true }))) {
             Write-ToLog -Message:('Script finished successfully; Log file location: ' + $jcAdmuLogFile) -Level Verbose
+            Write-ToProgress -ProgressBar $Progressbar -PercentComplete 100 -Status "Migration Completed Succesfully" -form $isForm
             Write-ToLog -Message:('Tool options chosen were : ' + "`nInstall JC Agent = " + $InstallJCAgent + "`nLeave Domain = " + $LeaveDomain + "`nForce Reboot = " + $ForceReboot + "`nUpdate Home Path = " + $UpdateHomePath + "`nAutobind JC User = " + $AutobindJCUser) -Level Verbose
-            if ($displayGuiPrompt) {
+            if (!$isForm) {
```

Comment:
I believe there's an unintended regression here:

When you migrate with the production version `2.6.8` we do not display a message post-migration. This version of the module displays the message box even if the script is invoked from CLI.

![Screenshot 2024-05-23 at 12 43 00 PM](https://github.com/TheJumpCloud/jumpcloud-ADMU/assets/54448601/a3da7695-9913-4511-b5ce-542c6dfa5455)
![Screenshot 2024-05-23 at 12 44 22 PM](https://github.com/TheJumpCloud/jumpcloud-ADMU/assets/54448601/cf81736c-53bc-4218-b9b9-40e196e7606c)